Source/Particles/Collision/BinaryCollision/ElasticCollisionPerez.H

```diff
@@ -15,23 +15,26 @@
 #include <AMReX_Random.H>
 
 
-/** \brief Prepare information for and call
- *        UpdateMomentumPerezElastic().
+/** Prepare information for and call UpdateMomentumPerezElastic().
+ *
+ * @tparam T_index type of index arguments
+ * @tparam T_R type of floating point arguments
+ * @tparam SoaData_type type of the "struct of array" for the two involved species
  * @param[in] I1s,I2s is the start index for I1,I2 (inclusive).
  * @param[in] I1e,I2e is the stop index for I1,I2 (exclusive).
- * @param[in] I1 and I2 are the index arrays. They determine all elements that will be used.
- * @param[in,out] u1 and u2 are the velocity arrays (u=v*gamma),
- *                they could be either different or the same,
- *                their lengths are not needed,
- * @param[in] w1 and w2 are arrays of weights.
- * @param[in] q1 and q2 are charges. m1 and m2 are masses.
- * @param[in] T1 and T2 are temperatures (Joule)
+ * @param[in] I1, I2 the index arrays. They determine all elements that will be used.
```

Comment:
```suggestion
 * @param[in] I1,I2 the index arrays. They determine all elements that will be used.
```

Looks like you removed those spaces in most places, so I'm guessing we have to remove them, is that correct?

Reply:
Yes, no spaces for these commas, otherwise Doxygen does not parse it properly - absolutely correct, thx for catching!